### PR TITLE
Add Stack 2.13.0.1 pre-release to GHCup metadata

### PR DESCRIPTION
**Pull request by mpilgrem**
See #120 for background.

### CODE REVIEW

File: ghcup-prereleases-0.0.7.yaml [hunk 1]
```diff
@@ -1372,6 +1373,46 @@ ghcupDownloads:
...
+          Linux_Alpine:
+            unknown_versioning: *stack-21301-64
+        A_ARM64:
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlUri: https://github.com/commercialhaskell/stack/releases/download/rc/v2.13.0.1/stack-2.13.0.1-linux-aarch64.tar.gz
+              dlHash: 67e6cea50dba52dbba39204605bf2c33154c88bee7f1494791a4f59589d64427
+              dlSubdir:
+                RegexDir: "stack-.*"
+        Darwin:
```
Comment (hasufell) [inline ghcup-prereleases-0.0.7.yaml:1409]: Indent

**Comment by wz1000** [comment]
https://github.com/haskell/ghcup-metadata/actions/runs/6314752240

**Comment by hasufell** [comment]
> https://github.com/haskell/ghcup-metadata/actions/runs/6314752240

you need to point it to `ghcup-prereleases-0.0.7.yaml`

**Comment by wz1000** [comment]
Hopefully https://github.com/haskell/ghcup-metadata/actions/runs/6314823904 should be the final word in this.

**Comment by wz1000** [comment]
Resigned and merged.

**Pull request merged**